src/api/Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'escape_utils'
 # to sanitize HTML/CSS
 gem 'sanitize'
 # as authorization system
-gem 'pundit'
+gem 'pundit', '>= 2.0.0.beta1'
```

Comment:
2.0.0 was released - see depfu's request

Reply:
great, i will update it, thanks @coolo